cf/irsa-api-role.yaml

```diff
@@ -76,7 +76,7 @@ Resources:
                   - !Sub "arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/experiments-${Environment}"
                   - !Sub "arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/plots-tables-${Environment}"
                   - !Sub "arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/samples-${Environment}"
-        - PolicyName: !Sub "can-download-objects-from-result-buckets-${Environment}"
+        - PolicyName: !Sub "can-download-objects-from-s3-buckets-${Environment}"
```

Comment:
I changed this to have the policy name be more accurate, but in the case this might cause issues let me know and I can change it.